sdk/qrcode/android/java/com/google/cardboard/sdk/qrcode/CardboardParamsUtils.java

```diff
@@ -102,39 +109,52 @@ private UriToParamsStatus(int statusCode, @Nullable byte[] params) {
   /** URI of original cardboard QR code. */
   // TODO(b/170471141): Wifi connection shouldn't be required for Cardboard Viewer v1.0.
   private static final Uri URI_ORIGINAL_CARDBOARD_QR_CODE =
-          new Uri.Builder()
-                  .scheme(HTTPS_SCHEME)
-                  .authority(URI_HOST_GOOGLE_SHORT)
-                  .appendEncodedPath(URI_PATH_CARDBOARD_HOME)
-                  .build();
+    new Uri.Builder()
+      .scheme(HTTPS_SCHEME)
+      .authority(URI_HOST_GOOGLE_SHORT)
+      .appendEncodedPath(URI_PATH_CARDBOARD_HOME)
+      .build();
 
   /**
    * Converts a bytes array with an URI string to a Uri object.
    * Obtains the physical parameters of a Cardboard headset from the Uri (as bytes) by
-   * calling {@code createFromUri}.
+   * calling {@code getParamsFromUriString}.
    * Then writes the device parameters to a predefined storage location calling {@code
    * writeDeviceParams()}.
    *
-   * @param byteArrUri A bytes buffer with the URI to read the parameters from.
+   * @param uriAsBytes A bytes buffer with the URI to read the parameters from.
    * @param context The current Context. It is generally an Activity instance or wraps one or an
    *        Application. It is used to write to storage.
    */
-  private static void saveParamsFromUri(byte[] byteArrUri, Context context){
-    String strUri = new String(byteArrUri);
-    Uri uri = Uri.parse(strUri);
-    UriToParamsStatus uriToParamsStatus = getParamsFromUri(uri, new UrlFactory());
-    writeDeviceParams(uriToParamsStatus.params, context);
+  private static void saveParamsFromUri(byte[] uriAsBytes, Context context){
+    String uriAsString = new String(uriAsBytes);
+    UriToParamsStatus uriToParamsStatus = getParamsFromUriString(uriAsString, new UrlFactory());
+    // If getParamsFromUri() does not return an OK status don't write the device params
+    if (uriToParamsStatus.statusCode != STATUS_OK) {
```

Comment:
Consider removing the recently added import on top by replacing `STATUS_OK` by `UriToParamsStatus.STATUS_OK` here.